File: Dev/Frontend/LambdaPP_WebApp_Alpha_3.0/WildTask/src/app/logic/models/Rating.ts
import Employee from "./Employee"
import Task from "./Task"

export default class Request 
{
    rating_ID!: number
    emp_ID!: number
    reviewer_ID!: number
    
    employee!: Employee | null
    reviewer!: Employee | null

    task_ID!: number
    task!: Task | null

    rating_Rating!: number
    rating_Date!: string
    rating_Comment: string | null = null

    rating_Category!: number|null
}

File: Mobile/D2/LambdaPP_AndroidApp_D2/app/src/main/java/com/example/lambdapp_androidapp_d2/models/Rating.java
package com.example.lambdapp_androidapp_d2.models;

public class Rating {
    public int rating_ID;
    public int tEBridge_ID;
    public Task_Employee_Bridge task_Employee_Bridge;
    public int rating_Rating;
    public int reviewer_ID;
    public Employee reviewer;
    public String rating_Comment;
    public String rating_Date;
    public int task_ID;
    public int emp_ID;//this is the person being rated, their score
}
